### DISCUSSION TITLE
FEATURE REQUEST 'MOVE To & COPY To'

### DISCUSSION THREAD

**Discussion by ukmint7** [linuxmint/discussions#493] [Questions]
A feature 'MOVE to' or 'COPY to' for an object such as a file or folder to a new or different location by selecting it and pressing a short-cut key like M or 'Alt/Shift+M' (for Move to ) and C or 'Alt/Shift+C' (for Copy to) is highly desirable.

**Comment by umikali222** [comment]
If you want to move a file somewhere else without duplicating it, you can select a file (or folder) and press Ctrl + X to cut. Then you can just paste it with Ctrl + V, and it will move the file Asif you did Ctrl + C and Ctrl + V, but it would remove the 1st file, so you essentially moved the file somewhere else.

**Reply by ukmint7** [reply]
I know that. It is a very basic copy/cut paste method which everyone knows, for that you will have to go (drill down) to the destination folder to perform the 'paste' action (ctrl+v). But my idea is to expand the file explorer itself in order to move/copy the file/folder just by selecting the file/folder and click the shortcut 'M' or 'Ctrl/shift/alt+M' for 'move to', and  'C' or 'Ctrl/shift/alt+C' for 'copy to' operation. Hope you got it. Similar feature is available in the  'Faststone Image Viewer', a Windows program.

**Comment by mtwebster** [comment]
In Nemo, you can also add menu items to perform this:  Preferences->Context Menus:
![image](https://github.com/linuxmint/discussions/assets/262776/33cb5b0e-0c95-4c2b-a003-7da0631c5172)

**Reply by ukmint7** [reply]
Thanks a lot. I should have explored all the features in the help guide. But you saved my day!
